js/utils/salt.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Generate 16 bytes salt for bcrypt by seed
+ *
+ * Should return the same salt for the same seed
+ *
+ * @param  {string}   seed The seed for salt
+ * @return {string}        The resulted salt
+ */
```

Comment:
Because genSaltSync generates random salt. But client must send the same encrypted password each time, yes?